packages/perseus/src/util/interactive.js

```diff
@@ -10,15 +10,14 @@
 // TODO(emily): This file breaks our line length limits like nobody's business.
 // Figure out how to fix that.
 import $ from "jquery";
-import {vector as kvector} from "kmath";
-import {point as kpoint} from "kmath";
-import {line as kline} from "kmath";
+import {vector as kvector, point as kpoint, line as kline} from "kmath";
 import Raphael from "raphael";
 import _ from "underscore";
 
 // NOTE(jeresig): We include a special copy of jQuery vmouse so
 // that Perseus will continue to work in mobile native devices
 // (this should have no impact in the browser)
+// eslint-disable-next-line import/no-unassigned-import
 import "../jquery.mobile.vmouse.js";
```

Comment:
Oh. This looks like breaking convention for importing a jQuery plugin through our jQuery bundle.  Do we really have this file in two places in webapp?

Reply:
I sounds like this copy of vmouse.js is different from what's included in our jquery bundle in webapp.  We should probably verify that and if it isn't we can get rid of it.  Since this is currently the status quo I'm going to leave it for now.